[2025-05-08T11:32:43.318-0500] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-08T11:32:43.322-0500] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: request_operation.RealtimeData scheduled__2025-05-08T16:32:00+00:00 [queued]>
[2025-05-08T11:32:43.325-0500] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: request_operation.RealtimeData scheduled__2025-05-08T16:32:00+00:00 [queued]>
[2025-05-08T11:32:43.326-0500] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-05-08T11:32:43.339-0500] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): RealtimeData> on 2025-05-08 16:32:00+00:00
[2025-05-08T11:32:43.344-0500] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'request_operation', 'RealtimeData', 'scheduled__2025-05-08T16:32:00+00:00', '--job-id', '1039', '--raw', '--subdir', 'DAGS_FOLDER/operations.py', '--cfg-path', '/tmp/tmp16w7xf37']
[2025-05-08T11:32:43.345-0500] {standard_task_runner.py:105} INFO - Job 1039: Subtask RealtimeData
[2025-05-08T11:32:43.347-0500] {logging_mixin.py:190} WARNING - /home/ares/anaconda3/envs/iqoptions_env/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=23177) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-05-08T11:32:43.347-0500] {standard_task_runner.py:72} INFO - Started process 23208 to run task
[2025-05-08T11:32:43.370-0500] {task_command.py:467} INFO - Running <TaskInstance: request_operation.RealtimeData scheduled__2025-05-08T16:32:00+00:00 [running]> on host DESKTOP-0R7MK50.
[2025-05-08T11:32:43.418-0500] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='request_operation' AIRFLOW_CTX_TASK_ID='RealtimeData' AIRFLOW_CTX_EXECUTION_DATE='2025-05-08T16:32:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-08T16:32:00+00:00'
[2025-05-08T11:32:43.418-0500] {taskinstance.py:732} INFO - ::endgroup::
[2025-05-08T11:32:56.629-0500] {timeout.py:68} ERROR - Process timed out, PID: 23208
[2025-05-08T11:32:56.632-0500] {taskinstance.py:3313} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/ares/anaconda3/envs/iqoptions_env/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 763, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ares/anaconda3/envs/iqoptions_env/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 734, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ares/anaconda3/envs/iqoptions_env/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ares/anaconda3/envs/iqoptions_env/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 424, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ares/anaconda3/envs/iqoptions_env/lib/python3.12/site-packages/airflow/decorators/base.py", line 266, in execute
    return_value = super().execute(context)
                   ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ares/anaconda3/envs/iqoptions_env/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 424, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ares/anaconda3/envs/iqoptions_env/lib/python3.12/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ares/anaconda3/envs/iqoptions_env/lib/python3.12/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ares/anaconda3/envs/iqoptions_env/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ares/iqoptions_bot/dags/operations.py", line 24, in RealtimeData
    API = ConectionApi(gmail=gmail, password=password)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ares/iqoptions_bot/dags/utils/extraction.py", line 21, in __init__
    self.API.connect()
  File "/home/ares/anaconda3/envs/iqoptions_env/lib/python3.12/site-packages/iqoptionapi/stable_api.py", line 103, in connect
    check, reason = self.api.connect()
                    ^^^^^^^^^^^^^^^^^^
  File "/home/ares/anaconda3/envs/iqoptions_env/lib/python3.12/site-packages/iqoptionapi/api.py", line 842, in connect
    check_websocket, websocket_reason = self.start_websocket()
                                        ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ares/anaconda3/envs/iqoptions_env/lib/python3.12/site-packages/iqoptionapi/api.py", line 785, in start_websocket
    while True:
          ^^^^
  File "/home/ares/anaconda3/envs/iqoptions_env/lib/python3.12/site-packages/airflow/utils/timeout.py", line 69, in handle_timeout
    raise AirflowTaskTimeout(self.error_message)
airflow.exceptions.AirflowTaskTimeout: Timeout, PID: 23208
[2025-05-08T11:32:56.635-0500] {taskinstance.py:1226} INFO - Marking task as FAILED. dag_id=request_operation, task_id=RealtimeData, run_id=scheduled__2025-05-08T16:32:00+00:00, execution_date=20250508T163200, start_date=20250508T163243, end_date=20250508T163256
[2025-05-08T11:32:56.660-0500] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-05-08T11:32:56.700-0500] {local_task_job_runner.py:266} INFO - Task exited with return code 2
[2025-05-08T11:32:56.710-0500] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-08T11:32:56.720-0500] {local_task_job_runner.py:245} INFO - ::endgroup::
